crates/nu-command/src/math/round.rs

```diff
@@ -80,6 +80,16 @@ impl Command for SubCommand {
 }
 
 fn operate(value: Value, head: Span, precision: Option<i64>) -> Value {
+    // We treat int values as float values in order to avoid code repetition in the match closure
+    let value = if let Value::Int { val, span } = value {
+        Value::Float {
+            val: val as f64,
```

Comment:
This will cause a loss of precision for larger integers that exceed `2^53` but the question is how relevant this is in practice to implement a separate solution via integer division by `10^(-precision)` followed by multiplication as the naive implementation (or whatever exists as the most efficient algorithm in the decimal system)

Reply:
I don't really understand why the second implementation wouldn't cause a loss of precision, but the first one would.

Reply:
In case you have e.g.  `2^60 - 2` this would not accurately fit in the f64 mantissa and would loose more precision than removing the last decimal with  `math round -1`.